src/lib/skilling/skills/fletching/fletchables/arrows.ts

```diff
@@ -0,0 +1,87 @@
+import itemID from '../../../../util/itemID';
+import { Fletchable } from '../../../types';
+
+const Arrows: Fletchable[] = [
+	{
+		name: 'Headless arrow',
+		id: itemID('Headless arrow'),
+		level: 1,
+		xp: 1,
+		inputItems: { [itemID('Arrow shaft')]: 1, [itemID('Feather')]: 1 },
```

Comment:
i think transforming all of these to [this](https://github.com/gc/oldschoolbot/blob/0fce12db1a7fd72a4dd728a067f49b198ac92054/src/lib/skilling/skills/crafting/craftables/dragonhide.ts#L99) format would make it more readable too